### PR TITLE
added closeio create_activity_call method, updated testing-stub

### PR DESCRIPTION
**Pull request by n-zav**

### CODE REVIEW

File: closeio/contrib/testing_stub.py [hunk 1]
```diff
@@ -265,6 +265,17 @@ def create_activity_email(self, lead_id, **kwargs):
         emails[lead_id].append(email)
 
     @parse_response
+    def create_activity_call(self, lead_id, **kwargs):
```
Comment (syphar) [inline closeio/contrib/testing_stub.py]: the method should have the same arguments in the live api and the stub

**Comment by syphar** [comment]
looks good feature wise, @codingjoe can deploy it and create a version

**Comment by syphar** [comment]
or i'll do it later this day

**Comment by codingjoe** [comment]
Please add test for new method in end2end tests

File: closeio/contrib/testing_stub.py [hunk 2]
```diff
@@ -246,16 +246,16 @@ def delete_lead(self, lead_id):
...
 
     @parse_response
-    def create_activity_email(self, lead_id, **kwargs):
+    def create_activity_email(self, **kwargs):
         kwargs.setdefault('status', 'draft')
 
         emails = self._data('activity_emails', {})
 
+        email = kwargs
+        lead_id = email['lead_id']
```
Comment (codingjoe) [inline closeio/contrib/testing_stub.py]: `assert lead_id == email['lead_id']`

File: closeio/contrib/testing_stub.py [hunk 3]
```diff
@@ -246,16 +246,16 @@ def delete_lead(self, lead_id):
         del leads[lead_id]
 
     @parse_response
-    def create_activity_email(self, lead_id, **kwargs):
+    def create_activity_email(self, **kwargs):
```
Comment (codingjoe) [inline closeio/contrib/testing_stub.py]: deprecate signature
`warning.warn('The argument "lead_id" has be deprecated in favour of ...", DeprecationWarning)`

File: closeio/contrib/testing_stub.py [hunk 4]
```diff
@@ -265,6 +265,18 @@ def create_activity_email(self, lead_id, **kwargs):
         emails[lead_id].append(email)
 
     @parse_response
+    def create_activity_call(self, **kwargs):
+
```
Comment (codingjoe) [inline closeio/contrib/testing_stub.py]: no new line ;)

**Pull request merged**